### ISSUE TITLE
AcceptConditionFormPlugin is not compatible with current djangocms-text-ckeditor

### ISSUE THREAD

**Issue by rfleschenberg** [awesto/django-shop#504]
https://github.com/awesto/django-shop/blob/master/shop/cascade/checkout.py#L280

```
html_content = plugin_tags_to_user_html(html_content, context, placeholder)
```

```
TypeError: plugin_tags_to_user_html() takes 2 positional arguments but 3 were given
```

**Comment by rfleschenberg** [comment]
The breaking change was introduced in djangocms-text-ckeditor 3.2.1.

**Comment by jrief** [comment]
In release 0.10 this has been fixed by upgrading to djangocms-text-ckeditor version 3.4

**Comment by jrief** [comment]
fixed in version 0.10
